docs/source/operations/howto/sso-oidc.rst

```diff
@@ -0,0 +1,77 @@
+******************************
+Set up SSO with OpenID Connect
+******************************
+
+Substra supports external user management through OpenID Connect (OIDC). It is done per-backend, so each organization can use their own user provider and cohabit on the same network.
+
+OIDC users cannot be created as admins (you'll still need to create admin accounts as normal, through ``addAccountOperator.users``), and will all be assigned to a single channel.
+
+
+Set up the provider
+===================
+
+First, you'll need to set up an OIDC client at an identity provider (IP -- or OpenID provider, OP).
+
+The only claims substra need are ``openid email``, which any provider should be able to support. Allow ``<BACKEND URL>/oidc/callback`` as a redirect URI.
+
+Get your **provider URL**. Appending ``/.well-known/openid-configuration`` to this URL should return a JSON description of the provider's capabilities, which Substra will use for much of the configuration (otherwise, you can set endpoints by hand under ``oidc.provider.endpoints``.
```

Comment:
```suggestion
Get your **provider URL**. Appending ``/.well-known/openid-configuration`` to this URL should return a JSON description of the provider's capabilities, which Substra will use for much of the configuration. Otherwise, you can set endpoints by hand under ``oidc.provider.endpoints``.
```